lib/logic/player/content.dart

```diff
@@ -841,7 +956,8 @@ class ContentUtils {
         throw UnimplementedError();
       }
       assert(() {
-        contentPick<Song, void>(
+        // See contentPick documentation for why we need this. 
+        contentPick<Content, void>(
```

Comment:
```suggestion
        contentPick<Song, void>(
```

This assert currently throws, because the handler for Content throws `UnimplementedError`.
```dart
======== Exception caught by gesture ===============================================================
The following UnimplementedError was thrown while handling a gesture:
UnimplementedError

When the exception was thrown, this was the stack: 
#0      contentPick (package:sweyer/logic/player/content.dart:77:7)
#1      new ContentTuple.<anonymous closure> (package:sweyer/logic/player/content.dart:171:9)
#2      new ContentTuple (package:sweyer/logic/player/content.dart:178:8)
#3      FavoritesControl.toggleFavoriteCurrentSong (package:sweyer/logic/player/favorites.dart:124:21)
#4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:989:21)
#5      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:198:24)
#6      TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:608:11)
#7      BaseTapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:296:5)
```

Reply:
The same applies to the other asserts like this.